cdk/diagram.png.dot

```diff
@@ -0,0 +1,267 @@
+digraph "Diagram" {
```

Comment:
I'm wondering if this file should go in .gitignore.  I say this because it's got your local home directory path in there.  Generated maybe?